File: client/src/app/shared/validators/articleValidator.ts
import { Validator } from 'fluentvalidation-ts';
import { Article } from '../types/article';

export class ArticleValidator extends Validator<Article> {
    constructor() {
      super();
  
      this.ruleFor('title').notEmpty();

      this.ruleFor('content').notEmpty();
    }
  }

File: server/WebAPI/Types/ArticleValidator.cs
using FluentValidation;

namespace WebApi.Types
{
  public class ArticleValidator : AbstractValidator<ArticleAggregate>
  {
    public ArticleValidator()
    {
      RuleFor(article => article.title).NotEmpty();
      RuleFor(article => article.content).NotEmpty();
    }
  }
}
